src/cmdlib.sh

```diff
@@ -383,7 +383,23 @@ EOF
         # Because right now rpm-ostree doesn't look for .repo files in
         # each included dir.
         # https://github.com/projectatomic/rpm-ostree/issues/1628
-        cp "${workdir}"/src/config/*.repo "${tmp_overridesdir}"/
+        repos_found=0
+        if [ -n "$(find "${workdir}"/src/config -maxdepth 1 -name '*.repo' -print -quit)" ]; then
+            repos_found=1
+            echo "DEBUG: src/config *.repo found"
+            cp "${workdir}"/src/config/*.repo "${tmp_overridesdir}"/
+        fi
+        
+        if [ -n "$(find "${workdir}"/src/extrepos -maxdepth 1 -name '*.repo' -print -quit)" ]; then
+            repos_found=1
+            echo "DEBUG: extrepos *.repo found"
+            cp "${workdir}"/src/extrepos/*.repo "${tmp_overridesdir}"/
+        fi
+
+        if [ ${repos_found} == 0 ]; then
+            echo "ERROR: no *.repo files were found"
+            exit 1
+        fi
```

Comment:
Minor: this code block is gated on `if [ -d "${overridesdir}" ] || [ -d "${ovld}" ]; then` which will always be true (because e.g. `cosa init` creates the `overrides/` dir, and we have `overlay.d` in the src config repo). But I think to be explicit/correct we should still add another condition there for if `src/extrepos` was provided.

Reply:
To clarify, are you suggesting that we remove the if statement which checks if `src/config` has any `.repo` files? In the RHCOS case, wouldn't this cause builds to fail since there are no `.repo` files in `openshift/os`?

Reply:
I'm suggesting we add `|| [ -d "${workdir}/src/extrepos" ]` to the higher-level if-statement I quoted.